exporters/holstebro_helpers.py

```diff
@@ -0,0 +1,811 @@
+
+    return is_leader_unit
+
+
+def _is_cm_unit(ou):
+    is_cm_unit = False
+
+    if ou['parent']['parent'] != None and ou['parent']['name'].count(SETTINGS['imports.holstebro.leaders.common_management_name']) == 1:
+        is_cm_unit = True
+
+    return is_cm_unit
+
+
+def find_org_manager(mh, node):
+    """
+    Searches for a manager for the given node.
+    Returns the manager directly associated with the node
+    OR the enherited manager from higherlevel
+    If no managers at all in organisation, the empty
+    dict is returned
+    """
+
+    if node == None:
+        return {'uuid': ''}
+
+    new_manager = mh.read_ou_manager(node.name, True)
+
+    if new_manager:
+        return new_manager
+    elif node.depth != 0:
+        return find_org_manager(mh, node.parent)
+    else:
+        return {'uuid': ''}
+
+
+class HolstebroHelper(object):
+    def __init__(self, mh):
+        self.mh = mh
+        self.manager_info = self._get_manager_types()
+        self.default_responsibility = self._get_default_responsibility()
+
+    def _get_manager_types(self):
+        org_uuid = self.mh.read_organisation()
+        manager_types = self.mh._mo_lookup(org_uuid, 'o/{}/f/manager_type')
+        manager_levels = self.mh._mo_lookup(org_uuid, 'o/{}/f/manager_level')
+        manager_responsibility = self.mh._mo_lookup(
+            org_uuid, 'o/{}/f/responsibility')
+
+        return_dict = {'manager_types': {},
+                       'manager_levels': {},
+                       'manager_responsibility': {}}
+
+        for mt in manager_types['data']['items']:
+            return_dict['manager_types'].update({mt['name']: mt})
+
+        for ml in manager_levels['data']['items']:
+            return_dict['manager_levels'].update({ml['name']: ml})
+
+        for mr in manager_responsibility['data']['items']:
+            return_dict['manager_responsibility'].update({mr['uuid']: mr})
+        # TODO: Add check for SETTINGS['imports.holstebro.leaders.responsibility'] in responsibility
+
+        return return_dict
+
+    def _get_default_responsibility(self):
+        default_responsibility = SETTINGS['imports.holstebro.leaders.responsibility']
+
+        # TODO: check all availresponsibilities against default and assign this
+        # for responsibility in self.manager_info['manager_responsibility']:
+        #    if responsibility == default_responsibility:
+        #        break
+        return default_responsibility
+
+    def _get_org_level(self, ou):
+        if ou['org_unit_level']['name'] == 'NY5-niveau':
+            return self.manager_info['manager_levels']['Direktør']['uuid']
+
+        elif ou['org_unit_level']['name'] == 'NY4-niveau':
+            return self.manager_info['manager_levels']['Chef']['uuid']
+
+        elif ou['org_unit_level']['name'] == 'NY3-niveau':
+            return self.manager_info['manager_levels']['Leder']['uuid']
+
+        else:
+            return self.manager_info['manager_levels']['Leder']['uuid']
```

Comment:
Ligesom ved `find_org_manager` ville jeg her droppe `elif` for `if` og helt droppe `else` tilsidst.
Derudover ville jeg læse `self.manager_info['manager_levels']` ud til en variabel først, så det er tydeligt at vi kigger på det samme element hver gang. Jeg ville gøre det samme med `ou` opslaget.
Endelig ville jeg droppe `elif for NY3-niveau` da den returnere det samme som `else` casen, og vi dermed altså blot kan falde igennem.

Dermed fås:
```suggestion
        ou_level_name = ou['org_unit_level']['name']
        manager_levels = self.manager_info['manager_levels']
        if ou_level_name == 'NY5-niveau':
            return manager_levels['Direktør']['uuid']
        if ou_level_name == 'NY4-niveau':
            return manager_levels['Chef']['uuid']
        return manager_levels['Leder']['uuid']
```